edit.go

```diff
@@ -261,6 +290,16 @@ func (v *View) writeRune(x, y int, ch rune) error {
 		chr:     ch,
 	}
 
+	// prevent text misplace when inserting
+	// fix problem for a|bc enter 烫烫 => a烫b烫c
+	for i := 1; i < w; i++ {
+		v.lines[y][x+i] = cell{
+			fgColor: v.FgColor,
+			bgColor: v.BgColor,
+			chr:     '\x00',
+		}
+	}
```

Comment:
This makes sense, but I'm starting to think that we should maybe consider a more "global" fix. 

One question regarding this I have is: Do the runes/characters display correctly in the terminal? I did not use wide characters before (or at all), so I don't really know.

The reason for question is, that in the `tcell` repository, there is a demo program for wide characters and the way it displays them is by using "combine" parameter in `tcell.SetContent()` function.
https://github.com/gdamore/tcell/blob/a4c1216c52b909a90c04d2caf0c2c8a344704311/_demos/unicode.go#L83
Unfortunatelly we do not have this in our `tcellSetCell()` function implemented, so I'm not sure if it displays ok or not (or maybe those characters which you use are not affected?).

@mjarkk it might be a good idea to look into this also for simple `Fprint` usage with `view`, not just for editing.